front/src/modules/ui/table/options/components/TableViewsDropdownButton.tsx

```diff
@@ -125,16 +117,21 @@ export const TableViewsDropdownButton = ({
     [setViewEditMode],
   );
 
-  const handleDeleteViewButtonClick = useCallback(
-    (event: MouseEvent<HTMLButtonElement>, viewId: string) => {
-      event.stopPropagation();
+  const handleDeleteViewButtonClick = useRecoilCallback(
```

Comment:
what the advantage of using a useRecoilCallback over useCallback here? To me, the usecase is when we want to read a piece of state without taking dependency on it through react dependencies watch. Here, as we don't use snapshot + we are not doing a transaction with multiple set(), I'm not sure to see what's the benefit

Reply:
Done, I've changed it back to `useCallback`.